components/ListView/ListItemImages.js

```diff
@@ -43,10 +46,43 @@ class ListItemImages extends React.Component {
     this.props.setModalDeleteImageVisible(true);
   }
 
+  handleLogsShow() {
+    this.setState(prevState => ({ logsExpanded: !prevState.logsExpanded, fetchingLogs: !prevState.logsExpanded }));
+    fetchComposeLogApi(this.props.listItem.id).then(
+      logs => {
+        this.setState({ logsContent: logs, fetchingLogs: false });
+      },
+      () => {
```

Comment:
Normally we would want to catch the exception thrown by the API here and print the message. However the error message from the API is useless, so at this point this doesn't make any sense.

Reply:
Yeah this is fine.